src/components/LoginComponent/LoginComponent.jsx

```diff
@@ -48,11 +48,15 @@ class LoginComponent extends Component {
   }
 
   render() {
+    if (this.props.loginError === true){
+      alert("Email and password combination to not match.");
```

Comment:
Alert initially works when my password is wrong. But as I try to delete so I can re-enter it, the alert keeps popping up with every delete key pressed. The very last letter doesn't delete so I can't re-enter password.